addons/im_livechat/static/src/embed/common/thread_model_patch.js

```diff
@@ -72,6 +72,8 @@ patch(Thread.prototype, {
         if (this.type !== "livechat") {
             return super.imgUrl;
         }
-        return url(`/im_livechat/operator/${this.operator.id}/avatar`);
+        return url(`/im_livechat/operator/${this.operator.id}/avatar`, {
+            unique: this.operator.write_date,
+        });
```

Comment:
If we didn't have the issue with non-persisted member, we could just remove this route and use the standard avatar route, this is annoying.